spring-cloud-stream-binder-kinesis/src/test/java/org/springframework/cloud/stream/binder/kinesis/provisioning/KinesisStreamProvisionerTests.java

```diff
@@ -257,4 +258,40 @@ private static DescribeStreamResult describeStreamResultWithShards(
 				.withHasMoreShards(Boolean.FALSE));
 	}
 
+	@Test(expected = ResourceNotFoundException.class)
```

Comment:
Please, don't use this JUnit feature.
We prefer AssertJ's `assertThatThrownBy()` instead.